http/protocolversioning.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>Test HTTP's versioning scheme</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+[
+  "HTTP/0.7 280",
```

Comment:
HTTP/0 is effectively HTTP/0.9, which doesn't have a status code. So, this should fail in some fashion.